src/main/java/com/growup/pms/auth/controller/dto/request/OauthLoginRequest.java

```diff
@@ -0,0 +1,16 @@
+package com.growup.pms.auth.controller.dto.request;
+
+import com.growup.pms.auth.service.dto.oauth.OauthUserLoginCommand;
+import lombok.Builder;
+
+@Builder
+public record OauthLoginRequest(String code)
+{
+    public OauthUserLoginCommand toCommand() {
+        return  OauthUserLoginCommand.builder()
+                .code(code)
+                .build();
+    }
+}
+
```

Comment:
Nit: 컨벤션에 맞게 수정해주시면 감사하겠습니다! :cat: 

```suggestion
@Builder
public record OauthLoginRequest(String code) {

    public OauthUserLoginCommand toCommand() {
        return  OauthUserLoginCommand.builder()
                .code(code)
                .build();
    }
}
```

Reply:
엇...마음이 급해서 놓쳤네요...바로 수정하겠습니다!
리뷰 감사드립니당